src/components/DisplayField/index.tsx

```diff
@@ -68,17 +69,45 @@ export const DisplayField: React.FC<DisplayFieldProps> = ({
   }
 
   if (Number.isFinite(value)) {
-    displayText = new Intl.NumberFormat(i18n.language, {
+    displayValue = new Intl.NumberFormat(i18n.language, {
       useGrouping: false
     }).format(Number(value));
   }
 
   if (isMoment(value)) {
-    displayText = value.format(format);
+    displayValue = value.format(format);
   }
 
   if (Array.isArray(value)) {
-    displayText = value.join(', ');
+    displayValue = value.join(', ');
+  }
+
+  const isUrl = (candidate: string) => {
+    const lowerCandidate = candidate.toLowerCase();
+    const protocols = ['http://', 'https://', 'ftp://', 'file://', 'mailto:'];
+    if (protocols.some(protocol => lowerCandidate.startsWith(protocol))) {
+      return true;
+    }
+    // windows (network) paths
+    if (candidate.startsWith('\\\\')) {
```

Comment:
shouldn't this be '////' ?

Reply:
Windows uses backslashes. But there should be only two not four. I fixed that in 959d914